### ISSUE TITLE
Rollos lassen sich nicht schließen

### ISSUE THREAD

**Issue by gerd2005** [thkl/hap-homematic#612]
Hallo,

wenn ich über mein iPad oder iPhone Siri sage, dass ein Rollo geschlossen werden soll, erscheint die folgende Rückmeldung und das Roll öffnet sich.

![8435C743-D914-4116-B93C-8C8DB348BB9D](https://user-images.githubusercontent.com/72799068/216837548-4c73515c-9be8-4944-8ab2-b6c1f78bee73.jpeg)

Wenn ich den gleichen Befehl über meinen HomePod mini absetze, wird das Roll geschlossen.

Hat jemand das gleiche Problem?

Viele Grüße
Gerd

**Comment by blackbasket** [comment]
Nein, macht, was es soll...

**Comment by gerd2005** [comment]
Hallo,

heute ist auch wieder was seltsames passiert, siehe Bild. Hier erscheint sogar, was Siri versteht, öffnet aber das Rollo. Ich bin echt ratlos, woran kann es nur liegen? Ich habe gerade das neue Apple Update installiert. Leider bringt es keine Besserung. Das Seltsame ist, dass auf den iPhones und iPads das Problem auftritt. Auf den HomePods funktioniert alles einwandfrei.
Ich verstehe das nicht.

![21221B10-0E35-4681-8DA5-FA95BB290EF1](https://user-images.githubusercontent.com/72799068/218757074-b1c820d7-f61c-45da-b44e-f95f48380137.jpeg)

**Comment by gerd2005** [comment]
Hallo,

hat wirklich niemand das Problem?
Wenn ihr sagt "schließe das Rollo XY", wird dieser Befehl tatsächlich ausgeführt.?

Ich behelfe mir jetzt damit, dass ich sage "Fahre das Rollo XY" herunter. Das hilft zwar, ist aber trotzdem dämlich, weil der Request von Siri trotzdem "Schließe das Rollo XY" ist.

**Comment by blackbasket** [comment]
Gerade getestet... geht ohne Probleme...

**Comment by gerd2005** [comment]
Sehr komisch. Im Internet findet man vereinzelte Hinweise von anderen Nutzern, dass die das gleiche Problem haben. 
Aber hast du irgendeine Idee woran das liegen könnte?

**Comment by blackbasket** [comment]
Hmm, ggf, weil dein Rollo "Rollo" heißt uns meine "Rollo links" etc... das könnte die Zuordnung für Siri vereinfachen...

**Comment by blackbasket** [comment]
Ich sage normalerweise einfach "Rollo xy schließen" und nicht "schließe das Rollo xy"... wobei xy für den Raum oder das explizite Rollo stehen kann...

**Comment by gerd2005** [comment]
Danke für den Tipp.

Ich habe ein Rollo, das habe ich "Rollo 2" benannt. Mit diesem teste ich das.

Bei Befehl: "Rollo 2 schließen" passiert genau das, was passieren soll. Das Rollo schließt sich.
Sage ich: "schließe ich Rollo 2" fährt das Rollo hoch.

Das kann doch alles irgendwie nicht sein....

VG
gerd

**Comment by blackbasket** [comment]
Absurd... wird dir denn jeweils der aktuelle Status in HomeKit und CCU korrekt angezeigt? Welchen Aktor nutzt du?

**Comment by gerd2005** [comment]
Ja, wirklich....
Ja, der Status wird korrekt angezeigt.
Ich nutze überall den HMIP-FROLL.

**Comment by blackbasket** [comment]
Ok, da kann ich nichts zu sagen... ich nutze Somfy RTS über nanoCUL mit CuXD ;)

**Comment by Jpp909** [comment]
Ich kann das Problem bestätigen:
**"Schliesse Rolladen xy"  öffnet ihn. "Rolladen xy schliessen" schließt ihn wirklich.** 

Auch diese Befehle funktionieren:
- Fahre Rolladen runter / hoch
- Mache Rolladen zu
- Rolladen auf 0%
Eine Diskussion dazu gab es auch im Bosch Forum.

Ich benutze einen Raspberry 3 mit HM-LC-Bl1PBU-FM und HAP.  Das Problem tritt auf, wenn ich ein iPhone (IOS 17.0.2), eine Apple Watch (10.0.1) oder iPad (17.0.2) benutze. Wobei ich mich noch nicht getraut habe, auf die neuste Home Version zu gehen. 

**Das Problem tritt nicht auf, wenn ich ein älteres iPhone mit IOS 15.7.9 benutze.**
Mit anderen Worten Siri hat es mal richtig gemacht!

**Comment by gerd2005** [comment]
@Jpp909 
Hast du auch einen HomePod?

**Comment by Jpp909** [comment]
Nein, nur iPhone, iPad Pro, Watch. Das heißt auch, dass ich nicht von außen Apple Home benutzen kann. Siri geht nur im gleichen WLAN. Sonst benutze ich @Home und bei Bedarf Wireguard auf der Fritzbox.
